python/az/aro/azext_aro/_params.py

```diff
@@ -60,45 +60,45 @@ def load_arguments(self, _):
                    validator=validate_version_format)
 
         c.argument('pod_cidr',
-                   help='CIDR of pod network. Must be a minimum of /18 or larger.',
+                   help='CIDR of pod network. Must be a minimum of /18 or larger. [Default: 10.128.0.0/14]',
                    validator=validate_cidr('pod_cidr'))
         c.argument('service_cidr',
-                   help='CIDR of service network. Must be a minimum of /18 or larger.',
+                   help='CIDR of service network. Must be a minimum of /18 or larger. [Default: 172.30.0.0/16]',
                    validator=validate_cidr('service_cidr'))
         c.argument('outbound_type',
                    help='Outbound type of cluster. Must be "Loadbalancer" (default) or "UserDefinedRouting".',
```

Comment:
can we make this follow the same format for conveying the default values? Something like:

> Outbound type of cluster. Allowed values: Loadbalancer, UserDefinedRouting. [Default: Loadbalancer]